### PR TITLE
Update taxonomy documentation

### PR DESCRIPTION
**Pull request by koetsier**
trello: https://trello.com/c/pzbqjfTU/159-spike-investigate-impact-on-education-world-and-transport-tagging-interfaces-in-whitehall

- Few spelling errors
- Added information about the new phases
- Added information about the visible_to_departmental_editors flag

### CODE REVIEW

File: source/manual/taxonomy.html.md [hunk 1]
```diff
@@ -54,15 +54,22 @@ content item for this guidance document][example-guidance].
...
 The level one taxons are associated with the GOV.UK home page through
-the `level_one_taxons` link type.
+the `root_taxon` link type. The GOV.UK home page in turn has a 
+corresponding reverse link of link type `level one taxons`.
 
-This is the content item for the top-level "Education" taxon:
+This is the content item for the GOV.UK home page with all level one
+taxons listed.
+
+[https://www.gov.uk/api/content/] [govuk-taxon]
```
Comment (tomsabin) [inline source/manual/taxonomy.html.md]: The space between the square brackets is preventing this from being rendered as a link, but also the `govuk-taxon` doesn't have a reference like other links in the bottom of this page

File: source/manual/taxonomy.html.md [hunk 2]
```diff
@@ -98,6 +105,20 @@ a facet:
...
 [https://www.gov.uk/api/search.json?filter_part_of_taxonomy_tree=c58fdadd-7743-46d6-9629-90bb3ccc4ef0&facet_taxons=1000&count=0](https://www.gov.uk/api/search.json?filter_part_of_taxonomy_tree=c58fdadd-7743-46d6-9629-90bb3ccc4ef0&facet_taxons=1000&count=0)
 
+## Visibility
+
+Editors can use Whitehall to tag content to the taxonomy. They can see
+both the published as well as the draft taxonomy. Only content tagged
+to the published taxonomy can be used on the live site.
+
+Individual branches can be hidden from Editors by clearing the 
+`visible_to_departmental_editors` flag on level one taxons.
```
Comment (tomsabin) [inline source/manual/taxonomy.html.md]: Perhaps mention that this can be done through an interface in Content Tagger?

Comment (koetsier) [reply source/manual/taxonomy.html.md]: fixed

File: source/manual/taxonomy.html.md (hunk 1 above)
Comment (koetsier) [reply source/manual/taxonomy.html.md]: fixed

**Pull request merged**